eo/Dockerfile

```diff
@@ -1,3 +1,16 @@
-FROM birdhouse/pavics-jupyter-base:0.0.2
+FROM birdhouse/pavics-jupyter-base:0.1.0
 
-RUN echo "test" > /eo-img-test
\ No newline at end of file
+# must update as root, because of a permission error with pip
+USER root
```

Comment:
Oh I think maybe this needs another `chown` right before switching to user Jenkins on the base image https://github.com/bird-house/pavics-jupyter-base/blob/02d7560856a633d0affc2b0f214dd731d502ec12/Dockerfile#L17

Hopefully no need to switch back and forth between user root and jenkins.

Reply:
I am not sure here. I don't think a chown is required for the /opt/conda folder. The 'all' permission set for that folder in the base image remains unchanged here, so no need for adjustment. 
I've checked too in a specific image (eo), and it seemed to be fine. I was able to install a package, so it means user Jenkins still has the permission to manipulate the conda folder.

Reply:
So user Jenkins have permissions for conda but not pip? !  That's pretty weird, since pip is from the same conda environment.  What kind of error you get if you do not switch to root above?

Reply:
I think you're right! I can actually remove the `USER root` and `USER jenkins` steps. Since the conda folder has received 'all' permissions in the base image, the jenkins user is now able to do the update here.
I can also remove the `umask 0000` since we are running the update as jenkins now.

In the base file, we are running the conda create/update as root, and we have to use umask, so that the birdy env is created with 'all' permissions and the jenkins user can be able to use the conda extension later.

It is not necessary anymore here though. I tested it too and it the conda extension still seems to work!

Reply:
@tlvu I have now gotten the error back. I think I had just tested with conda dependencies in my yml file. The error happens when we put packages under pip dependencies in the yml file. It seems to be a PermissionError on a temporary requirements file created during the process: 
`PermissionError: [Errno 13] Permission denied: '/condaenv.bd1q6fly.requirements.txt'`

The jenkins user doesn't seem to have permission on that location, hence why the USER steps were there, so the conda update is made as root to avoid this permission error. I will put these steps back in the dockerfile here.

Reply:
That's unfortunate.  Put this explanation into your commit message so we have a trace.  You've seen how useful it was later with our archeological search in the PAVICS-e2e-workflow-tests repo :D